src/main/java/kostka/moviecatalog/service/runtimeconfig/RuntimeConfigService.java

```diff
@@ -0,0 +1,54 @@
+package kostka.moviecatalog.service.runtimeconfig;
+
+import kostka.moviecatalog.dto.RuntimeConfigDto;
+import kostka.moviecatalog.entity.RuntimeConfig;
+import kostka.moviecatalog.exception.InvalidDtoException;
+import kostka.moviecatalog.repository.RuntimeConfigRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class RuntimeConfigService {
+    private final RuntimeConfigRepository runtimeConfigRepository;
+
+    @Autowired
+    public RuntimeConfigService(
+            final RuntimeConfigRepository runtimeConfigRepository) {
+        this.runtimeConfigRepository = runtimeConfigRepository;
+    }
+
+    public void updateRuntimeConfiguration(final RuntimeConfigDto dto) {
+        validateNewValue(dto);
+
+        RuntimeConfig runtimeConfig = null;
+        List<RuntimeConfig> runtimeConfigList = runtimeConfigRepository.findAll();
+
+        if (runtimeConfigList.isEmpty()) {
```

Comment:
Idea: if liquibase or something similar is used in this project, I would add some changelog to insert default row into a database (if you can't delete the runtime config via UI or something). Than you don;t need to case about it in code (less states in the application = less tests are needed for proper coverage = probably less code = less probability to introduce a bug).

Reply:
I have no liquibase, but I can add this to the data.sql file for initial database import

Reply:
Urcite, tady to chce mit jen jednu.